src/Blocks/BlockTorch.h

```diff
@@ -192,13 +192,18 @@ class cBlockTorchHandler:
 
 	virtual bool CanBeAt(cChunkInterface & a_ChunkInterface, const Vector3i a_RelPos, const cChunk & a_Chunk) override
 	{
-		auto Face = MetaDataToBlockFace(a_Chunk.GetMeta(a_RelPos));
-		auto NeighborRelPos = AddFaceDirection(a_RelPos, Face, true);
+		if (!cChunkDef::IsValidHeight(a_RelPos.y - 1))
+		{
+			return false;
```

Comment:
I believe that torches can be at Y=0 since they can attach to the side of a block, right?

Reply:
Same goes for levers/buttons/tripwirehook I guess. While also a check for attached edge face from meta could be done to validate not being placed on "-1" but I think the overhead for this also edge case is ridiculous.